source/assets/sass/components/_sass-syntax-switcher.scss

```diff
@@ -110,7 +110,7 @@
   // syntax switcher is exactly equal to two lines, so there's no visual jitter
   // when switching between syntaxes.
   pre + pre {
-    margin-top: 22px;
+    margin-top: 1rem;
```

Comment:
@mirisuzanne @stacyk This seems to fix the issue (also on the old site) with differing heights when multiple code blocks are stacked. See: https://github.com/oddbird/sass-site/pull/17/files#r1134278706

Reply:
@jgerigmeyer are the tabs supposed to work in this branch?

Reply:
@stacyk Yes! They're working for me locally and on the deploy preview...

Reply:
ok, it is working for me now... I will update my comments